circle.yml

```diff
@@ -9,3 +9,13 @@ test:
     - nvm install 7.4.0 && npm run test
   post:
     - chmod +x ./bin/codacy-coverage.js && cat ./coverage/lcov.info | node ./bin/codacy-coverage.js
+
+deployment:
+  npm:
+    branch: master
+    commands:
+      - echo -e "_auth = $NPM_TOKEN\nemail = $NPM_EMAIL" > .npmrc
+      - npm version minor -m "Release version %s"
+      - git push --tags
```

Comment:
I don't think that we can push tags from circleCI as it is (the deploy key seems to be read-only)